codecs/mozjpeg/Makefile

```diff
@@ -27,10 +27,12 @@ $(OUT_WASM): $(CODEC_OUT)
 		-I $(CODEC_DIR) \
 		${CXXFLAGS} \
 		${LDFLAGS} \
+		-mexec-model=reactor \
 		-o $@ \
 		enc/mozjpeg_enc.cpp \
 		$(CODEC_OUT)
-	$(BINARYEN_PREFIX)/bin/wasm-opt -O3 -o $(OUT_WASM) $(OUT_WASM)
+	$(BINARYEN_PREFIX)/bin/wasm-opt -O4 -o $(OUT_WASM) $(OUT_WASM)
+	$(WABT_PREFIX)/bin/wasm-strip $(OUT_WASM)
```

Comment:
I think wasm-opt can do everything that wasm-strip can do so you shouldn't need both tools

Reply:
@sbc100 Oh, I missed those flags. `--strip-debug --strip-dwarf --strip-producers --strip-target-features` seems to achieve the same thing. Cool!